tasks/create_storage_domain.yml

```diff
@@ -1,10 +1,18 @@
 ---
 - name: Create hosted engine local vm
   block:
-  - name: Wait until all interfaces wake up
-    wait_for:
-      timeout: 150
-      delegate_to: "{{ he_ansible_host_name }}"
+  - name: Wake up the storage interface
+    command: ip link set '{{ he_storage_if }}' up
+    delegate_to: "{{ he_ansible_host_name }}"
+    when: he_domain_type == "glusterfs" or he_domain_type == "nfs"
+  - name:  Wait for the storage interface to be up 
+    command: cat /sys/class/net/'{{ he_storage_if }}'/operstate
```

Comment:
please use `ip -j link show {{ he_storage_if }}` instead. (unless we want to cherry-pick this to 4.3 -el7)